dotCMS/src/integration-test/java/com/dotcms/experiments/business/web/ExperimentWebAPIImplIT.java

```diff
@@ -483,4 +541,57 @@ public void isUserIncludedWithVariants() throws DotDataException, DotSecurityExc
             ExperimentDataGen.end(experiment);
         }
     }
+
+    /**
+     * Method to test: {@link ExperimentWebAPIImpl#isUserIncluded(HttpServletRequest, HttpServletResponse, List)}
+     * When:
+     * - You have one experiment running with 100% of traffic allocation and without targeting and called the method
+     * it should return this experiments, and also include it into the included list, the excluded list should be empty.
```

Comment:
this experiment? experiments?

Reply:
done https://github.com/dotCMS/core/pull/23410/commits/1ae4364cc2de81cefd382676d1e076f53ea02a6b#diff-7c1d4e7a1777c7c52ce86a8e605b66ccc8b79b6297d3c9e3f3d33059226ea3b3R549